### DISCUSSION TITLE
Is there any way to only log user activity?

### DISCUSSION THREAD

**Discussion by arall** [spatie/laravel-activitylog#980] [Q&A]
I have an app that performs a lot of activity on cron jobs, commands and automated jobs. 

I would like to avoid logging when running those so, in short, only log user activity.

Is there any way to only log when the user session is active (`Auth::check()` for example)?

**Comment by Gummibeer** [comment]
You can disable logging by default or on runtime and enable it in the auth middleware for example.
Or any other middleware/place you have access to the needed information.
